webservices/resources/reports.py

```diff
@@ -105,11 +134,18 @@ def build_query(self, committee_id=None, committee_type=None, **kwargs):
         if kwargs.get('is_amended') is not None:
             query = query.filter(reports_class.is_amended == kwargs['is_amended'])
 
+        query = filters.filter_range(query, kwargs, self.filter_range_fields)
         return query, reports_class, reports_schema
 
     def _resolve_committee_type(self, committee_id=None, committee_type=None, **kwargs):
-        if committee_id is not None:
-            query = models.CommitteeHistory.query.filter_by(committee_id=committee_id)
+        #if kwargs.get('committee_id') is not None and len(kwargs.get('committee_id')) > 0:
+        if committee_id is not None or kwargs.get('committee_ids'):
+            comm_id = (
+                committee_id
+                if committee_id is not None
+                else kwargs.get('committee_ids')[0]
```

Comment:
Ah I see. Thanks for explaining. Ordinarily I'd be cool with just doing the simpler solution for now, but in this case it would require making a new UI component in the form of a typeahead filter that only allows for one committee. 

👍  thanks for merging. I'll follow up with folks on Monday.